controllers/tools/index.js

```diff
@@ -40,10 +39,7 @@ router.route('/').get((req, res) => {
         { label: 'View a list of all published pages', href: '/tools/pages' },
         { label: 'View micro-survey results', href: '/tools/survey-results' },
         { label: 'View feedback results', href: '/tools/feedback-results' },
-        { label: 'View recent materials order stats', href: '/tools/order-stats' },
-        { label: 'View the Reaching Communities & Partnerships data dashboard', href: TOOLS_DATASTUDIO_URL },
```

Comment:
These are overkill and are a bit of scope creep on what this index page should do, which is just list internal tools URLs.

Reply:
Fair.